src/explore-education-statistics-admin/src/modules/find-statistics/components/KeyStatistics.tsx

```diff
@@ -0,0 +1,191 @@
+import { EditableContentBlock } from '@admin/services/publicationService';
+import { releaseContentService } from '@admin/services/release/edit-release/content/service';
+import Button from '@common/components/Button';
+import styles from '@common/modules/find-statistics/components/SummaryRenderer.module.scss';
+import { EditingContext } from '@common/modules/find-statistics/util/wrapEditableComponent';
+import {
+  AbstractRelease,
+  Publication,
+} from '@common/services/publicationService';
+import React, { useContext, useEffect, useState } from 'react';
+import EditableKeyStatTile from './EditableKeyStatTile';
+import KeyIndicatorSelectForm from './KeyIndicatorSelectForm';
+
+export interface KeyStatisticsProps {
+  release: AbstractRelease<EditableContentBlock, Publication>;
+  setRelease: (
+    newRelease: AbstractRelease<EditableContentBlock, Publication>,
+  ) => void;
+  isEditing?: boolean;
+}
+
+const KeyStatistics = ({
+  release,
+  setRelease,
+  isEditing,
+}: KeyStatisticsProps) => {
+  const [keyStats, setKeyStats] = useState<
+    | AbstractRelease<EditableContentBlock, Publication>['keyStatisticsSection']
+    | undefined
+  >(undefined);
```

Comment:
Don't think you need `undefined` here - by not supplying any arguments, that's undefined